paychmgr/simple.go

```diff
@@ -492,6 +567,8 @@ func (ca *channelAccessor) addFunds(ctx context.Context, channelInfo *ChannelInf
 	return &mcid, nil
 }
 
+// TODO func (ca *channelAccessor) freeFunds(ctx context.Context, channelInfo *ChannelInfo, amt, avail types.BigInt) (*cid.Cid, error) {
```

Comment:
Want before landing, needed for stuff to work well with how we fund partial retrievals (which we fund like a full retrieval)

Reply:
((apparently we don't have good market hooks for this; can be done later))